src/test/testData/Parsing/declarations/func-def/func-def-10/func-def-10.bal

```diff
@@ -0,0 +1,2 @@
+function foo(int a, Person p) returns Student {
+}
```

Comment:
```suggestion
}

```
Lets make sure to add the trailing new line in the eof. [1]

[1] - [https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)